changelog.json

```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "3.1.3-beta2": [
+      "[Added] Enable menu option to Force-push branches that have diverged - #15521",
+      "[Added] Add menu option to Fetch the current repository at any time - #15520",
+      "[Added] Support VSCodium as an external editor - #15348. Thanks @daniel-ciaglia!",
+      "[Fixed] Prevent closing the GitHub Desktop while it's being updated - #15416",
+      "[Fixed] Notifications are shown only when they are relevant to the current repository - #15487",
+      "[Fixed] Disable reorder, squashing, cherry-picking while an action of this type is in progress. - #15468",
+      "[Fixed] Fix repository change indicator not visible if selected and in focus - #15475. Thanks @angusdev!",
+      "[Fixed] Close 'Resolve conflicts before Rebase' dialog will not disable menu items - #15471. Thanks @angusdev!",
+      "[Fixed] Tooltips are positioned properly if mouse is not moved - #15462. Thanks @angusdev!",
+      "[Fixed] Fix tooltips of long commit author emails not breaking properly - #15460. Thanks @angusdev!",
+      "[Fixed] Clone repository progress bar no longer hidden by repository list - #15465. Thanks @angusdev!",
+      "[Fixed] Fix commit shortcut (Ctrl/Cmd + Enter) while amending a commit - #15445",
+      "[Improved] Pull request preview dialog width and height is responsive - #15500"
```

Comment:
Are these all auto-generated? I was expecting to see references to the original issue (where possible) instead of references to the PR that addressed them. For example, the note about #15520 should actually point to #7805 🤔

Reply:
No. The auto generation failed. I never realized that the numbers pointed to issues.

Reply:
Yeah, the script is supposed to try and extract the original issue from a PR and, if not possible, it just leaves the PR number. I'd swear I saw it working properly in previous versions after my changes from #14727 😕

Reply:
It wasn't the issue portion that failed. It failed to retrieve the PRs all together. It did that the time before too.. I just got a new access token in case maybe I set it up wrong last time... maybe next time it will work.

Reply:
I went through and manually pointed to related issues if available